docker/nginx/default.conf

```diff
@@ -1,13 +1,26 @@
 server {
-  listen 80 default_server;
+  listen 80 default_server reuseport;
 
   keepalive_timeout 70;
 
   index index.php;
 
-  location / {
-    root /usr/share/nginx/html;
-    try_files $uri /index.php$is_args$args;
+  root /usr/share/nginx/html;
+  try_files $uri /index.php$is_args$args;
+
+  # security headers
+  add_header X-XSS-Protection          "1; mode=block" always;
```

Comment:
Consider checking this doc, especially the recommendation in the first note.

IMO we should remove this and add a task to check CSP configuration later.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection

Here is the doc, since I forgot to link it somehow :eyes:

Reply:
Agreed! I'll remove it.